File: sadlab/mainapp/codes/c/011202299.c
#include <stdio.h>

int main() {
    double num1, num2, num3;
    scanf("%lf", &num1);
    scanf("%lf", &num2);
    scanf("%lf", &num3);
    double sum = num1 + num2 + num3;
    printf("%.2f\n",sum);
    return 0;
}

File: sadlab/mainapp/codes/java/011202299.java
import java.util.Scanner;

public class AddThreeNumbers {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        double num1 = scanner.nextDouble();
        double num2 = scanner.nextDouble();
        double num3 = scanner.nextDouble();
        double sum = num1 + num2 + num3;
        System.out.println(sum);
        scanner.close();
    }
}